contrib/snowball/template.py

```diff
@@ -0,0 +1,42 @@
+pkgname = "snowball"
+pkgver = "2.2.0"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+# check_utf8 check_iso_8859_1 check_iso_8859_2 check_koi8r
+# all have missing sub-targets
+make_check_target = "check_stemtest"
+hostmakedepends = ["gmake", "perl"]
+pkgdesc = "Snowball rule-based stemmin alorithms"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "BSD-3-Clause"
+url = "https://snowballstem.org/index.html"
+source = f"https://github.com/snowballstem/snowball/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "425cdb5fba13a01db59a1713780f0662e984204f402d3dae1525bda9e6d30f1a"
+env = {"LD_LIBRARY_PATH": "./"}
```

Comment:
what is this one for? libstemmer?

Reply:
yes, the tests can’t find libstemmer otherwise.  Other distros have a libstemmer sub package